platform/ios/src/MGLMapView.mm

```diff
@@ -1655,9 +1653,9 @@ - (void)handleQuickZoomGesture:(UILongPressGestureRecognizer *)quickZoom
     {
         CGFloat distance = [quickZoom locationInView:quickZoom.view].y - self.quickZoomStart;
 
-        CGFloat newZoom = log2f(self.scale) + (distance / 75);
+        CGFloat newZoom = MAX(log2f(self.scale) + (distance / 75), _mbglMap->getMinZoom());
```

Comment:
Note: it’s still necessary to clamp this to `minZoom` because we want to bail if this gesture won’t change the zoom/camera.